code/modules/antagonists/starfurybc/starfurybc.dm

```diff
@@ -0,0 +1,82 @@
+/datum/antagonist/starfuryop
+	name = "Syndicate Operative"
+	roundend_category = "starfury battlecruiser operatives"
+	antagpanel_category = "Starfury Operative"
+	job_rank = ROLE_TRAITOR
+	antag_hud_type = ANTAG_HUD_OPS
+	antag_hud_name = "synd"
+	antag_moodlet = /datum/mood_event/badass_antag //focused
+	var/datum/team/starfuryop/sbccrew
```

Comment:
Please add documentation to make this dmdoc compliant.

Reply:
It's stil WIP, but what do you mean by "dmdoc compliant"?

Reply:
And yeah, moodlet will be "focused" instead.